environment_setup/setup_grsim_arch.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~#
+# UBC Thunderbots Arch Linux grSim Setup
+#
+# This script will install all the required libraries and dependencies to run
+# grSim. This only includes the simulator (grSim), and not anything related to
+# our AI. See the setup_software.sh script for that.
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~#
+
+# Save the parent dir of this script since we want to return here
+# so that commands are run relative to the location of this script. This
+# helps prevent bugs and odd behaviour if this script is run through a symlink
+# or from a different directory.
+CURR_DIR=$(dirname -- "$(readlink -f -- "$BASH_SOURCE")")
+cd $CURR_DIR
+
+
+# Install required packages
+echo "================================================================"
+echo "Downloading and installing required packages."
+echo "================================================================"
+
+# The list of packages required to build and run grSim
+# See the github repo for the list of dependencies
+# https://github.com/RoboCup-SSL/grSim/blob/master/INSTALL.md
+
+host_software_packages=(
+    git
+    cmake
+    protobuf
+    qt5-base
+    protobuf
+    ode
+    boost
+)
+
+sudo pacman -Sy
+sudo pacman -S ${host_software_packages[@]} --noconfirm
```

Comment:
Looks like the only real difference between this setup script and the ubuntu one is the package names and the package manager? I don't feel super strongly about this because this is not a script we're officially supporting, but might be worth removing the duplication here.

Reply:
"removing duplication" as in removing packages that were already installed using the setup_software script or merging the two setup_grsim scripts together?

Reply:
merging the two scripts together.